app/controllers/import_users_controller.rb

```diff
@@ -70,7 +77,7 @@ def valid_input_file?(imported_users, file) # rubocop:disable all
     unless imported_users.first.keys == accepted_keys
       flash[:error] = 'Unable to import CSV file. Please ensure that the '\
         'first line of the file exactly matches the sample input (username, '\
```

Comment:
On second thought, we should double check that the example given on the page is correct (I know this probably falls under #520 but I'd say it also falls under this issue).